docs/components/stratosphere.md

```diff
@@ -0,0 +1,10 @@
+# Stratosphère
+Stratosphère allows customization of the Horizon OS and Switch kernel. It includes custom sysmodules that extend the kernel and provide new features. It also includes a reimplementation of the loader to hook important system actions.
+
+The sysmodules that Stratosphère includes are:
++ boot: This module boots the system and launches other processes.
```

Comment:
Contrary to what one might expect, `boot` doesn't actually launch any other processes. `boot` does some hardware initialization. `boot2` (launched by PM on stock firmware, included within PM on stratosphere) is the one that launches other processes.